src/tailwind-sx/src/sxGenerator.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import csstree from 'css-tree';
+import * as changeCase from 'change-case';
+
+const NOT_SUPPORTED_PROPS = new Set([
+  'grid-gap',
+  'grid-row-gap',
+  'grid-column-gap',
+  'pointer-events',
+]);
+const UNITLESS_PROPS = new Set(['line-height', 'font-size']);
+
+// eslint-disable-next-line flowtype/no-weak-types
+export default function generateSX(css: string | Buffer): Object {
```

Comment:
Would it be possible to avoid weak types in this file as well, please? We spend great effort migrating the codebase from this legacy type and I don't want to go back (we still allow `any` though). 😬 You can also replace it with `$FlowFixMe` type suppress type like so if you don't want to deal with it now:

```suggestion
export default function generateSX(css: string | Buffer): $FlowFixMe {
```